components/sst_importer/src/metrics.rs

```diff
@@ -107,6 +107,11 @@ lazy_static! {
         "Bucketed histogram of importer apply count",
         &["type"]
     ).unwrap();
+    pub static ref INPORTER_COMPACT_KEYS_COUNT: IntCounterVec = register_int_counter_vec!(
+        "tikv_import_compact_keys_count",
+        "Bucketed histogram of importer download keys from compacted SST files",
```

Comment:
Please correct this description. Obviously, it's not a histogram, and it's a counter for `compacted/filtered keys` not `download keys`.